src/webapp/src/components/Navbar/index.ts

```diff
@@ -1 +1,2 @@
 export * from './Navbar';
+export * from './constants';
```

Comment:
What about my other comment which is related to this, did you perhaps had the chance to think about it ?

Reply:
Do you mean the one where you mentioned this, right?

> Can we move the Navbar.ts code into the index.ts please? Maybe on a different PR

I created an issue to do it in a different PR, but I can do it here too. I was trying to keep it organized but it is true it is not a bid deal.

Reply:
Different Pr is better imo